pkg/interface/src/logic/reducers/hark-update.ts

```diff
@@ -6,19 +6,21 @@ import {
 } from "~/types";
 import { makePatDa } from "~/logic/lib/util";
 import _ from "lodash";
+import {StoreState} from "../store/type";
```

Comment:
linter asks that we put spaces between the brackets here — maybe start running eslint before committing?